Client/Assets/ContextMenu.js

```diff
@@ -77,17 +78,30 @@ function handleTouchMove(event) {
 function handleTouchEnd(event) {
   cancel();
 
-  removeEventListener("touchend", handleTouchEnd);
-  removeEventListener("touchmove", handleTouchMove);
+  window.removeEventListener("touchend", handleTouchEnd);
+  window.removeEventListener("touchmove", handleTouchMove);
```

Comment:
Just added `window` here to add clarity to the code. This is effectively a non-change.